doc/source/howtos/upgrade_guide.rst

```diff
@@ -81,16 +81,16 @@ To upgrade the cluster, follow these steps for each node:
 
    .. code-block:: bash
 
-      sudo mysqld --skip-grant-tables --user=mysql --wsrep-provider='none'
+      $ sudo mysqld --skip-grant-tables --user=mysql --wsrep-provider='none'
 
-   .. note:: As of |PXC| 5.7.6, the ``--skip-grant-tables`` option
-      is not required.
+   .. note::
 
-   .. note:: To prevent any users from accessing this node while performing
-      work on it, you may add `--skip-networking <https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_skip-networking>`_
+      To prevent any users from accessing this node while performing
+      work on it, you may add `--skip-networking
+      <https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_skip-networking>`_
       to the startup options and use a local socket to connect, or
-      alternatively you may want to divert any incoming traffic from your
-      application to other operational nodes.
+      alternatively you may want to divert any incoming traffic from
+      your application to other operational nodes.
 
 #. Open another session and run ``mysql_upgrade``.
```

Comment:
mysql_upgrade has been deprecated and actually does nothing now.  They will need to run mysqld on the datadir to upgrade.
Also, PXC will now run upgrade automatically if an old datadir is detected (this is now built into MySQL from 8.0.16).  In one sense, they don't need this step, but it is probably better in case the upgrade fails.